src/Structures/Page.php

```diff
@@ -405,6 +405,11 @@ public function private()
         return $this->entry()->private();
     }
 
+    public function status()
+    {
+        return optional($this->entry())->status();
+    }
+
```

Comment:
If we're not allowing filtering of navs, I think this method doesn't need to be added.